src/__tests__/__snapshots__/MissionsPage.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders Component correctly 1`] = `
+<table
+  className="container table table-striped table-bordered table-hover"
+>
+  <thead>
+    <tr>
+      <th>
+        Description
+      </th>
+      <th>
+        Status
+      </th>
+      <th>
+        Action
+      </th>
+    </tr>
+  </thead>
+  <tbody />
+</table>
```

Comment:
- [ ] one of your test is failing, kindly fix
![image](https://user-images.githubusercontent.com/66097371/202452225-e4c7cecd-ca70-4755-8366-8d1f161346de.png)